app/src/main/java/org/mozilla/firefox/vpn/apptunneling/data/AppTunnelingRepository.kt

```diff
@@ -69,7 +69,8 @@ class AppTunnelingRepository(
     }
 
     private fun ApplicationInfo.isSystemApp(): Boolean {
-        return (this.flags and ApplicationInfo.FLAG_SYSTEM) != 0
+        val systemFlag = ApplicationInfo.FLAG_SYSTEM or ApplicationInfo.FLAG_UPDATED_SYSTEM_APP
```

Comment:
nit: How about adding a comment to describe this is a defensive check to OR both flags.

Reply:
I moved this method to AppTunnelingUtils.kt, and add a comment as below

```
fun ApplicationInfo.isSystemApp(): Boolean {
    // For the apps on our devices, FLAG_SYSTEM is always set if FLAG_UPDATED_SYSTEM_APP is set.
    // Being uncertain if it's possible an app can have only FLAG_UPDATED_SYSTEM_APP being set, we
    // defensively check both bits here
    val systemFlag = ApplicationInfo.FLAG_SYSTEM or ApplicationInfo.FLAG_UPDATED_SYSTEM_APP
    return (this.flags and systemFlag) != 0
}
```